src/Components.tsx

```diff
@@ -14,10 +14,15 @@ export const ConnectPayouts = (): JSX.Element => {
 
 export const ConnectPaymentDetails = ({
   chargeId,
+  payment,
   onClose,
   visible = undefined,
 }: {
-  chargeId: string;
+  /**
+   * @deprecated
+   */
+  chargeId?: string;
+  payment?: string;
```

Comment:
Can we add a JSDOC comment here? I think that can help people understand if this can be a charge id, payment intent id...